### DISCUSSION TITLE
Dropdown in navbar does not behave as per example

### DISCUSSION THREAD

**Discussion by Askancy** [saadeghi/daisyui#2244] [Q&A] [answered]
I'm trying to put the navbar in my project, but the dropdown doesn't behave like in the example, where am I going wrong?

Example: https://daisyui.com/components/navbar/#responsive-dropdown-menu-on-small-screen-center-menu-on-large-screen

Code: https://codepen.io/Askancy/pen/poQXpxG

**Comment by prudhvi-mallavarapu** [comment, marked as answer]
You are using 2.6.0 version for daisyui styles `https://cdn.jsdelivr.net/npm/daisyui@2.6.0/dist/full.css`. please check with the newer version once.
